community/kernel/src/main/java/org/neo4j/kernel/impl/api/index/IndexStoreView.java

```diff
@@ -44,10 +49,17 @@
      * @return a {@link StoreScan} to start and to stop the scan.
      */
     <FAILURE extends Exception> StoreScan<FAILURE> visitNodes( int[] labelIds, int[] propertyKeyIds,
```

Comment:
@lutovich yes maybe it's a subset but still think it's better then to have method that will recieve null in half of the cases